pytreex/block/read/conllu.py

```diff
@@ -98,5 +111,8 @@ def process_document(self, filename):
         else:
             for i in xrange(1,len(nodes)):
                 nodes[i].parent = nodes[parents[i]]
+            if len(comment):
+                root.set_attr('wild/comment', comment)
```

Comment:
More suitable would be zone than bundle (CoNLL-U currently supports only one zone, but we plan to extend it). How can I store a comment in zone (or in bundle)? I'm getting `AttributeError: 'Bundle' object has no attribute 'set_attr'`